src/Parser/Core/Modules/AlwaysBeCasting.js

```diff
@@ -109,7 +109,12 @@ class AlwaysBeCasting extends Module {
     cast,
     spellId
   ) {
-    const timeWasted = castStartTimestamp - (this.lastCastFinishedTimestamp || this.owner.fight.start_time);
+    //Iskalla: Inserted this so time wasted never goes negative, even if there's a haste buff not yet implemented
```

Comment:
This showing a negative dead gcd time is intended to reveal issues in the analysis. I would prefer leaving the "bug" in so that it continues to be easy to spot.